pom.xml

```diff
@@ -14,9 +14,9 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <grpc.version>1.39.0</grpc.version>
-    <protobuf.version>3.13.0</protobuf.version>
-    <dapr.proto.baseurl>https://raw.githubusercontent.com/dapr/dapr/v1.6.0-rc.3/dapr/proto</dapr.proto.baseurl>
+    <grpc.version>1.42.1</grpc.version>
+    <protobuf.version>3.17.3</protobuf.version>
```

Comment:
@artursouza  Is there any concern in changing the protobuf version here ?

Reply:
No. LGTM.